CONTRIBUTING.md

```diff
@@ -318,29 +318,29 @@ GCP project `computer-vision-team`, configure minikube and skaffold
     - `pluginsSettings`
     - `teamsAppSettings`
 
-   For example for the version `1.6.0` at `dev7`.
+   For example for the version `1.6.1` at `dev7`.
 
     ```yaml
     apiSettings:
       image:
         repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-teams-api
-        tag: v1.6.0.dev7
+        tag: v1.6.1.dev7
```

Comment:
Does 1.6.1.dev7 exist? Or are we setting bad configs that breaks our docs??

Maybe let's not change this file.